src/randomenv.cpp

```diff
@@ -60,10 +61,11 @@ void RandAddSeedPerfmon(CSHA512& hasher)
     // Seed with the entire set of perfmon data
 
     // This can take up to 2 seconds, so only do it every 10 minutes
-    static int64_t nLastPerfmon;
-    if (GetTime() < nLastPerfmon + 10 * 60)
-        return;
-    nLastPerfmon = GetTime();
+    static std::atomic<int64_t> last_perfmon{0};
+    int64_t last_time = last_perfmon.load();
+    int64_t current_time = GetTime();
```

Comment:
in commit ad09ad707e0ed6e3a537e1a204843792e5d06108:

`GetTime` is deprecated (see https://dev.visucore.com/bitcoin/doxygen/time_8cpp.html#a350f99e2a13df31f2afbd7f80ab21a5e). I guess you wanted to use `GetSystemTimeInSeconds`

Reply:
`GetTime<std::chrono::seconds>()` can't hurt here either, as far as I can tell

Reply:
Fixed. Also done elsewhere in the previous commit.